### DISCUSSION TITLE
Anyone know where is the cache file location after enabled "-- disk"?

### DISCUSSION THREAD

**Discussion by Noocn** [oobabooga/text-generation-webui#2053] [Q&A]
After enabling this, my C driver turns red. It did improve a little, but I cannot locate the cache file. It is not in the ..\AppData\Local\TEMP directory. Does anyone know where it might be located? Thank you very much

**Comment by mrdevolver** [comment]
Hi, I have the exact same problem. I wanted to run the program completely on my D drive including cache data and I trusted the --disk parameter would let the cache data store in the same directory with the program, but I was very unpleasantly surprised when I ran out of free space on C drive. Not cool. I tried to locate where the cache is stored using some methods which usually work, but unfortunately they did not work this time. Any help would be appreciated!

**Reply by Fleischkuechle** [reply]
try this C:\Users\Admin\.cache  
Im logged in as Admin in my system so you need to replace Admin with your logged in user. 
I ran into the same issue  i went into my user folder and searched for the biggest folder in there (I used this handy tool https://foldersize.sourceforge.net/)
this is a very small but super helpfull tool to see the biggest folder in a opend folder.
finally i found it
![grafik](https://github.com/oobabooga/text-generation-webui/assets/45604838/180d99a8-a2d4-4d8a-bb30-e7f0ec29f58a)
